docs/en/models/sam-2.md

```diff
@@ -113,6 +113,8 @@ The following table details the available SAM 2 models, their pre-trained weight
 
 | Model Type  | Pre-trained Weights                                                                   | Tasks Supported                              | Inference | Validation | Training | Export |
 | ----------- | ------------------------------------------------------------------------------------- | -------------------------------------------- | --------- | ---------- | -------- | ------ |
+| SAM 2 tiny  | [sam2_b.pt](https://github.com/ultralytics/assets/releases/download/v8.2.0/sam2_t.pt) | [Instance Segmentation](../tasks/segment.md) | ✅        | ❌         | ❌       | ❌     |
+| SAM 2 small | [sam2_l.pt](https://github.com/ultralytics/assets/releases/download/v8.2.0/sam2_s.pt) | [Instance Segmentation](../tasks/segment.md) | ✅        | ❌         | ❌       | ❌     |
```

Comment:
Looks like the pretrained weights strings are not correct here
```diff
- | SAM 2 tiny  | [sam2_b.pt](https://github.com/ultralytics/assets/releases/download/v8.2.0/sam2_t.pt) | [Instance Segmentation](../tasks/segment.md) | ✅        | ❌         | ❌       | ❌     |
+ | SAM 2 tiny  | [sam2_t.pt](https://github.com/ultralytics/assets/releases/download/v8.2.0/sam2_t.pt) | [Instance Segmentation](../tasks/segment.md) | ✅        | ❌         | ❌       | ❌     |
- | SAM 2 small | [sam2_l.pt](https://github.com/ultralytics/assets/releases/download/v8.2.0/sam2_s.pt) | [Instance Segmentation](../tasks/segment.md) | ✅        | ❌         | ❌       | ❌     |
+ | SAM 2 small | [sam2_s.pt](https://github.com/ultralytics/assets/releases/download/v8.2.0/sam2_s.pt) | [Instance Segmentation](../tasks/segment.md) | ✅        | ❌         | ❌       | ❌     |
```

Reply:
@Burhan-Q nice catch!

Reply:
Yes you're right!

Reply:
t and s now instead of b and l...

Reply:
@glenn-jocher can you update this as well? I feel I might break your workflow if I directly update it since you are working on the branch. :) Thanks!

Reply:
Oh wait, we've got all 4 it looks like @Laughing-q?

<img width="381" alt="Screenshot 2024-07-30 at 13 52 35" src="https://github.com/user-attachments/assets/c56296bb-f747-4047-9d2e-da461e53a7b3">

Reply:
@Burhan-Q can you update the table there for all four models? I see we have 4 assets uploaded:

<img width="785" alt="Screenshot 2024-07-30 at 13 53 38" src="https://github.com/user-attachments/assets/742a44d5-8a73-489e-af34-103c396310b0">

Reply:
@glenn-jocher yes we've got all four
![pic-240730-1954-22](https://github.com/user-attachments/assets/db3a00f9-f4bb-4a9f-aa4a-c4095b9c1dd0)

Reply:
@glenn-jocher @Burhan-Q ok updated